RIRImageResizing/Classes/NSObject/RIRResizeImageOperation.h

```diff
@@ -0,0 +1,32 @@
+//
+//  RIRResizeImageOperation.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import "RIRResizeImageOperationParameters.h"
```

Comment:
Classes should be forward declared in the .h whenever possible (except for Apple classes). Go ahead and forward declare `RIRResizeImageOperationParameters` in this .h, and import it in the .m if you need to.